github-event-service/pull-request.ts

```diff
@@ -0,0 +1,112 @@
+import * as firebase from 'firebase-admin';
+
+import {getFirebaseInstance} from './firebase-common';
+
+type DocumentReference = firebase.firestore.DocumentReference;
+
+/** The configuration for the branch manager. */
+export interface BranchManagerConfig {
+  enabled: boolean;
+  branches: Array<{
+    branch: string;
+    label: string;
+  }>;
+};
+
+/** 
+ * Limited implementation of the Github pull_request webhook event from
+ * https://developer.github.com/webhooks/#payloads
+ */
+export interface GithubPullRequestEvent { 
```

Comment:
Would it be a good idea to move those Github event definitions into a different file? So that this file is mainly about the logic to handle the incoming PR request event.

Reply:
True, I definitely need to do some thinking about the structure within the repo to figure out where this could go